VirtoCommerce.CustomerModule.Web/Scripts/blades/filter-detail.tpl.html

```diff
@@ -0,0 +1,77 @@
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="form-label">{{ 'customer.blades.filter-detail.labels.searchInVariations' | translate }}</label>
+                    <div class="form-input">
+                        <label class="form-label __switch">
+                            <input type="checkbox" ng-model="blade.currentEntity.searchInVariations">
+                            <span class="switch"></span>
+                        </label>
+                    </div>
+                </div>
+            </form>
+            <div class="form-group">
+                <div class="list __info">
+                    <div class="list-t">org1.com</div>
+                    <div class="list-descr">all contacts with word 'org1.com' in property values</div>
+                </div>
+                <div class="list __info">
+                    <div class="list-t">James Smith</div>
+                    <div class="list-descr">all contacts with words 'James' AND 'Smith' in property values</div>
+                </div>
+                <div class="list __info">
+                    <div class="list-t">Ale*</div>
+                    <div class="list-descr">use '*' to replace zero or more characters</div>
+                </div>
+
+                <div class="list __info">
+                    <div class="list-t">firstname:Alex,James</div>
+                    <div class="list-descr">all contacts where first name contains words 'Alex' or 'James'. </div>
+                </div>
+                <div class="list __info">
+                    <div class="list-t">!lastname:Smith</div>
+                    <div class="list-descr">use ! as NOT operator on filters</div>
+                </div>
+                <div class="list __info">
+                    <div class="list-t">membertype:Contact</div>
+                    <div class="list-descr">all contacts with the selected member type</div>
+                </div>
+                <div class="list __info">
+                    <div class="list-t">createddate:[2016-12-26 TO 2018-12-29]</div>
+                    <div class="list-descr">all contacts with creation period, between values including bounds </div>
+                </div>
+                <div class="list __info">
+                    <div class="list-t">createddate:(2016-12-26 TO 2018-12-29)</div>
+                    <div class="list-descr">all contacts with creation period, between values excluding bounds</div>
+                </div>
+                <div class="list __info">
+                    <div class="list-t">modifieddate:[2016-12-26 TO]</div>
+                    <div class="list-descr">where the modified date is greater or equal than '2016-12-26'</div>
+                </div>
+                <div class="list __info">
+                    <div class="list-t">@org1.com membertype:Contact</div>
```

Comment:
Empty result at the sample data (at least for me). Please use working example (without @org1.com)

Reply:
fixed